server/views/review/includes/licenceSummary.pug

```diff
@@ -1,5 +1,7 @@
 include additionalConditions
 
+-var validAddress = (!licenceStatus.decisions.curfewAddressRejected && !licenceStatus.decisions.bassAreaNotSuitable)
+
```

Comment:
Make the licence summary view follow the same display rules as the tasklist view